src/rpc.rs

```diff
@@ -82,6 +82,18 @@ impl Mission for RPC {
         Ok(Response::new(res))
     }
 
+    async fn get_airbases(
+        &self,
+        request: Request<GetAirbasesRequest>,
+    ) -> Result<Response<GetAirbasesResponse>, Status> {
+        let res: GetAirbasesResponse = self
+            .ipc
+            .request("getAirbases", Some(request.into_inner()))
+            .await
+            .map_err(|err| Status::internal(err.to_string()))?;
```

Comment:
I've simplified the API here part a bit, so we can shorten it to the following:

```suggestion
        let res: GetAirbasesResponse = self.request("getAirbases", request).await?;
```

Reply:
I made the change, I hope you don't mind

Reply:
Awesome, thanks!